tests/toolkit/matl_buy_policy_tests.cc

```diff
@@ -181,12 +182,49 @@ TEST_F(MatlBuyPolicyTests, MultiReqQuantize) {
   ASSERT_FLOAT_EQ(req->target()->quantity(), quantize);
 }
 
-// Tests that agent cycles between active and dormant cycles during a mock
-// sim, only buying when active.
-TEST_F(MatlBuyPolicyTests, ActiveDormant) {
-  int active = 2;
-  int dormant = 3;
-  int dur = 4;
+TEST_F(MatlBuyPolicyTests, DefaultFixedActiveDormant) {
+  using cyclus::QueryResult;
+  
+  FixedIntDist a_dist = FixedIntDist(1);
+  FixedIntDist d_dist = FixedIntDist(-1);
+  FixedDoubleDist size_dist = FixedDoubleDist(1.0);
+  
+  int dur = 2;
+  double throughput = 1;
+
+  cyclus::MockSim sim(dur);
+  cyclus::Agent* a = new TestFacility(sim.context());
+  sim.context()->AddPrototype(a->prototype(), a);
+  sim.agent = sim.context()->CreateAgent<cyclus::Agent>(a->prototype());
+  sim.AddSource("commod1").Finalize();
+
+  TestFacility* fac = dynamic_cast<TestFacility*>(sim.agent);
+
+  cyclus::toolkit::ResBuf<cyclus::Material> inbuf;
+  cyclus::toolkit::MatlBuyPolicy policy;
+  policy.Init(fac, &inbuf, "inbuf", throughput, &a_dist, &d_dist, &size_dist)
+        .Set("commod1").Start();
+
+  EXPECT_NO_THROW(sim.Run());
+
+  QueryResult qr = sim.db().Query("Transactions", NULL);
+  // confirm that transactions are only occurring during active periods
+  int first_cycle = qr.GetVal<int>("Time", 0);
+  EXPECT_EQ(0, first_cycle);
+  int second_cycle = qr.GetVal<int>("Time", 1);
+  EXPECT_EQ(1, second_cycle);
+
+  delete a;
+}
+
+TEST_F(MatlBuyPolicyTests, FixedActiveDormant) {
+  using cyclus::QueryResult;
+  
+  FixedIntDist a_dist = FixedIntDist(1);
+  FixedIntDist d_dist = FixedIntDist(1);
```

Comment:
I'm not sure what our best practice should be, but you could re-use these distributions if their identical, especially if they are `FixedIntDist`s
```suggestion
  FixedIntDist a_d_dist = FixedIntDist(1);
```
and pass `a_d_dist` for both the active and dormant distribution.